source/MapOutfitterPanel.cpp

```diff
@@ -244,7 +244,7 @@ void MapOutfitterPanel::DrawItems()
 				: storedInSystem == 1
 				? "One unit in storage"
 				: Format::Number(storedInSystem) + " units in storage";
-			Draw(corner, outfit->Thumbnail(), isForSale, outfit == selected,
+			Draw(corner, outfit->Thumbnail(), 0, isForSale, outfit == selected,
```

Comment:
This should use the virtual function, too.

Reply:
Wait which one?

Reply:
Did you see this comment? I'm not sure what virtual function you're referring to. If you refer to the ones that I added they refer to the selected/compare swizzles, so it seems inappropriate to use them here.

Reply:
Correct, they don't match up well. That suggests room for a "default swizzle" getter.

Reply:
How? It works for outfits but there is no default swizzle for ships if I understand you correctly.